source/api_documentation/network_integration/advertiser_campaigns/index.rst

```diff
@@ -403,6 +444,9 @@ Node Details
   * - YesOrNo
     - May have exactly 2 child nodes. If a keypress of 1 is made, the first child node is executed. If a kepyress of 2 is made, the second child node is executed. If speech recognition is enabled, the caller can also say "yes" for 1 and "no" for 2. At the end of the child list, this node type can also optionally have failover child nodes, designated by a node with a keypress_failover_type parameter (see example below).
 
+  * - RingGroup
+    - May not have any children. Forwards the call to a group of numbers that will be dialed in the order determined by the distribution method. If a destination plays a busy signal OR the ring_group_connect_timeout value is exceeded before the call is answered (or accepted), the next destination will be dialed. This node type also allows for simultaneous calling when it is enabled.
```

Comment:
> May not have any children

Is this a little confusing given `ivr_ringing_rollover_extensions` functionality exists?

Reply:
Valid point, but considering we're waiting to document `ivr_ringing_rollover_extensions`, saying it does allow a single child may be confusing as well. Thoughts?

Reply:
You could just not reference "children" at all